esx_service/vmdk_ops.py

```diff
@@ -688,15 +688,16 @@ def get_vol_path(datastore, tenant_name=None):
     # is created on <datastore>/DOCK_VOLS_DIR/tenant_uuid
     # a symlink <datastore>/DOCK_VOLS_DIR/tenant_name will be created to point to
     # path <datastore>/DOCK_VOLS_DIR/tenant_uuid
+
     dock_vol_path = os.path.join("/vmfs/volumes", datastore, DOCK_VOLS_DIR)
+    path = dock_vol_path
```

Comment:
nit: could be `path = dock_vol_path = os.path.join("/vmfs/volumes", datastore, DOCK_VOLS_DIR)`